Tests/Editor/Selection/SelectFacesByColor.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UObject = UnityEngine.Object;
+using NUnit.Framework;
+using UnityEngine.ProBuilder;
+using UnityEditor.ProBuilder;
+using UnityEditor;
+using UnityEngine.TestTools;
+
+
+namespace UnityEngine.ProBuilder.EditorTests.Selection
+{
+    class SelectFacesByColor
+    {
+        ProBuilderMesh[] selectables;
+        List<Color> faceColors;
+
+        void Setup(bool withColor = false)
```

Comment:
I'm not sure if this would work since you have an argument, but adding [SetUp] attribute would make it so this gets called automatically before each test

Reply:
Given the argument in the setup it will not work this time. But I will know for the future! Thank you!